content/lessen/wiskunde/rekenen_rationaal/breuken/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Wat is een breuk?"
+date: 2020-07-05T15:00:00+02:00
+weight: 2
+draft: true
+images: []
+tags: ["Breuk", "Rationale getallen", "Bewerkingen", "Waarom kan je niet delen door 0?"]
+description: "In deze les zoeken we uit wat een breuk precies is."
+---
+Je bent op een feestje met acht personen en het is tijd voor de taart :birthday:. Iedereen wil uiteraard een even groot
+stuk van de taart, dus wordt hij in acht gelijke stukken gesneden :cake:. Iedereen krijgt dus één achtste of $\frac{1}{8}$ van de taart.
+
+## Waar staat de teller?
+
+{{< svg "img/terminologie.svg" "Een breuk bestaat uit een breukstreep met daarboven de teller. Onder de breukstreep staat de noemer." >}}
+
+De **teller** *telt* het aantal delen dat je neemt. De **noemer** *benoemd* hoe groot het deel is :exploding_head:. Je kan onthouden: de **t**eller staat aan de **t**op! 
+
+| Je schrijft       | Je leest het als               |
+|-------------------|--------------------------------|
+| $\frac{a}{b}$     | a over b                       |
+
+## Hoe werkt een breuk?
```

Comment:
Deze paragraaf is misschien iets te veel "van de hak op de tak". Probeer om per paragraaf duidelijk af te lijnen waar ze over gaat. Dat zou je in één zin heel concreet moeten kunnen zeggen en misschien is "hoe en breuk werkt" iets te algemeen. Het lijkt me ook dat de eerste alinea's kunnen samengevoegd worden met de vorige paragraaf. Bv. de volgende paragrafen:

1. Een breuk bestaat uit een teller en een noemer -> Waar staan teller en noemer in een breuk en hoe kan je ze interpreteren?
2. Een breuk als deling -> Een breuk is eigenlijk een deling
3. Tekens in een breuk -> Een minteken kan je doorheen de breuk verschuiven. Twee mintekens mag je schrappen.
4. Gelijkwaardige en gelijknamige breuken -> Gelijkwaardige breuk zijn breuken die gelijk zijn aan elkaar, gelijknamige breuken zijn breuken met dezelfde noemer
5. Breuken, kommagetallen en procenten als rationale getallen -> Breuken, (rationale) kommagetallen en procenten zijn inwisselbaar. Ze zijn allemaal elementen van de rationale getallen.